src/label-extractor.js

```diff
@@ -2,7 +2,7 @@ module.exports.extractLabels = function (labelsString) {
     // Parses a list of labels. Each label can be of any length and will either end with a comma or be the end of the string.
     // Matches words (\w), whitespace characters (\s), dashes (-), plus signs (+), questions marks (\?), semi-colons (;), brackets (\[\]) and parenthesis (\(\))
     // Each match may are may not have a trailing comma (,?). If one exists, it is removed before appending it to the list
-    const regex = new RegExp(/([\w\s-+\?;\[\]\(\)]+,?)/, 'g')
+    const regex = new RegExp(/([\w\s-\/+\?;\[\]\(\)]+,?)/, 'g')
```

Comment:
👍🏼 Thank you.

Only thing is can we update the comment on line 3 to point out `/` now matches?

Reply:
Done :)